src/Inline/Element/AbstractInlineContainer.php

```diff
@@ -44,9 +39,9 @@ public function getChildren()
      */
     public function setChildren($contents)
     {
-        $this->children = $contents;
-        foreach ($contents as $content) {
-            $content->setParent($this);
+        $this->detachChildren();
+        foreach ($contents as $children) {
+            $this->appendChild($children);
```

Comment:
This seems to do the same thing as `replaceChildren`, so I'd be okay with removing this method and using the base `replaceChildren` method instead.

Actually, I think we could probably remove this class altogether and use `ContainerNode` instead.  Thoughts?